SuperGraphics/Block.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "Model.h"
+#include "Primitives.h"
+class Block : public Model
+{
+public:
+	Block() {
```

Comment:
We'll need to separate blocks from each other, BrownBlock should have its own class